DropManager/tests/DropManager-specs.js

```diff
@@ -0,0 +1,110 @@
+import {Layout, Cell} from '@enact/ui/Layout';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
+import Button from '../../Button';
+import Droppable, {Draggable, ResponsiveBox} from '../DropManager';
+
+describe('DropManager Specs', () => {
+	const allSlotNames = ['bottom', 'center', 'top'];
+	const DraggableCell = Draggable(Cell);
+	const containerShapes = {
+		bottom: {orientation: 'landscape', edges: {bottom: true}},
+		center: {orientation: 'portrait', edges: {top: false, bottom: false}},
+		top:    {orientation: 'landscape', edges: {top: true}}
+	};
+
+	const ResponsiveLayout = ResponsiveBox(({containerShape, ...rest}) => {
```

Comment:
It looks `ResponsiveLayout` and `containerShapes` are not required to test. It could be okay to leave them if they are useful to expand tests someday later. Please let me know the reason why these code are here.

Reply:
I am not sure I understand why ResponsiveBox is not required to test since it is part of DropManager. DropManager is quite complex and I may have not understood right. Could you please explain a bit? Thank you.

Reply:
Nevermind, it was my fault that I missed some code line in specs. Sorry! 😢